src/corehost/cli/fxr/fx_muxer.h

```diff
@@ -102,24 +103,29 @@ class fx_muxer_t
         const fx_ver_t& specified,
         bool patch_roll_fwd,
         roll_fwd_on_no_candidate_fx_option roll_fwd_on_no_candidate_fx);
-    static int read_framework(
+    static StatusCode read_framework(
         const host_startup_info_t& host_info,
         const fx_reference_t& override_settings,
         const runtime_config_t& config,
         fx_name_to_fx_reference_map_t& newest_references,
         fx_name_to_fx_reference_map_t& oldest_references,
         fx_definition_vector_t& fx_definitions);
+    static StatusCode resolve_frameworks_for_app(
```

Comment:
I'll work on a much bigger refactoring after this change - there are lot of methods in this class which don't need to be in the header at all. In fact the class should be broken up into smaller ones - most local to the cpp file - it would simplify the code even more.

Reply:
For now I'll leave it as is for consistency with the rest.